pkg/api/chunk_stream.go

```diff
@@ -67,6 +67,8 @@ func (s *Service) chunkUploadStreamHandler(w http.ResponseWriter, r *http.Reques
 			jsonhttp.BadRequest(w, "invalid batch id")
 		case errors.Is(err, errUnsupportedDevNodeOperation):
 			jsonhttp.BadRequest(w, errUnsupportedDevNodeOperation)
+		case errors.Is(err, postage.ErrBatchInUse):
+			jsonhttp.BadRequest(w, "batch is in use by another upload process")
```

Comment:
This message is repeated, make it an error, similar to `errUnsupportedDevNodeOperation`.